apps/meteor/client/views/account/featurePreview/AccountFeaturePreviewPage.tsx

```diff
@@ -105,15 +105,16 @@ const AccountFeaturePreviewPage = () => {
 					)}
 					{featuresPreview.length > 0 && (
 						<>
-							<Box
-								className={css`
-									white-space: break-spaces;
-								`}
-								pbe={24}
-								fontScale='p1'
-							>
-								{t('Feature_preview_page_description')}
-							</Box>
+							<FieldGroup marginBlockEnd={16}>
```

Comment:
`FieldGroup` and `Field` should be used to form fields, you can use `Box` instead